lib/general.js

```diff
@@ -83,12 +84,12 @@ checks.push(new OptionalFfmpegCommandCheck());
 class OptionalMjpegConsumerCommandCheck extends DoctorCheck {
   async diagnose () {
     const packageName = 'mjpeg-consumer';
-    const packageInfo = await getNpmPackageInfo(packageName);
-
-    if (packageInfo) {
-      return okOptional(`${packageName} is installed at: ${packageInfo.path}. Installed version is: ${packageInfo.version}`);
+    try {
+      await node.requirePackage(packageName);
+      return okOptional(`${packageName} is installed.`);
+    } catch {
+      return nokOptional(`${packageName} cannot be found.`);
```

Comment:
same as above
does it makes sense to extract package presence validation into a single function to avoid unnecessary code deuplication?